firefox-ios/Client/Frontend/Browser/MainMenu/Redux/MainMenuState.swift

```diff
@@ -69,14 +81,16 @@ struct MainMenuState: ScreenState, Equatable {
         currentTabInfo: MainMenuTabInfo?,
         submenuDestination: MainMenuDetailsViewType? = nil,
         navigationDestination: MenuNavigationDestination? = nil,
-        shouldDismiss: Bool = false
+        shouldDismiss: Bool = false,
+        accountData: AccountData? = nil
```

Comment:
should this be `nil` by default? Because that means that every reducer action where don't pass the account data in, we lose the account data, no?

Reply:
From my perspective, yes, it should be nil by default because I want to pass AccountData only on the viewDidLoad MainMenuActionType, also in newState(...) I am updating the UI only when I have AccountData.